src/FiniteHorizonPOMDPs.jl

```diff
@@ -1,5 +1,16 @@
 module FiniteHorizonPOMDPs
 
-greet() = print("Hello World!")
+using POMDPs
+using POMDPModelTools
 
-end # module
+import POMDPs: Policy, action
+
+export 
+    FiniteHorizonPolicy,
```

Comment:
I think `FiniteHorizonPolicy` is not a great name because there could be other types of policies for problems with finite horizons. Perhaps `FiniteHorizonValuePolicy` would be better.

Reply:
Done that.